### PR TITLE
[Framegear] Add support for `post_redirect`, `link` + icons

### PR DESCRIPTION
**Pull request by taycaldwell**
**What changed? Why?**
- Add support for `post_redirect`, `link` actions.
- Add external link icon for `post_redirect`, `link` buttons.

**Screenshot**
Screenshot 2024-02-27 at 4 58 17 PM

**Notes to reviewers**
N/A

**How has it been tested?**
Manually

### CODE REVIEW

**Comment by taycaldwell** [comment]
PR addresses some items listed in the following roadmap: https://github.com/coinbase/onchainkit/issues/146

File: framegear/app/page.tsx [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
 
 export default function Home() {
   return (
-    <div className="mx-auto flex flex-col items-center gap-8 pb-16">
-      <Header />
-      <div className={`max-w-layout-max grid w-full grid-cols-[5fr,4fr] gap-16`}>
-        <div className="flex flex-col gap-4">
-          <FrameInput />
-          <Frame />
+    <RedirectModalProvider>
```
Comment (Zizzamia) [inline framegear/app/page.tsx:10]: Oh Nice

File: framegear/components/Frame/Frame.tsx [hunk 2]
```diff
@@ -101,38 +103,58 @@ function FrameButton({
...
       className="border-button w-[45%] grow rounded-lg border bg-white px-4 py-2 text-black"
       type="button"
       onClick={handleClick}
-      disabled={button?.action !== 'post'}
+      disabled={isLoading || button?.action === 'mint'}
     >
       <span className="block max-w-full overflow-hidden text-ellipsis whitespace-nowrap">
         {children}
+        {button?.action === 'post_redirect' || button?.action === 'link' ? (
+          <ArrowTopRightIcon className="ml-1 inline" />
```
Comment (cnasc) [inline framegear/components/Frame/Frame.tsx:154]: We should probably pick a distinct icon for each case (not a blocker for this PR, can be done in future)

**Review by cnasc — APPROVED** [review]
LGTM, one nonblocker we can do in the future

**Pull request merged**